functions.php

```diff
@@ -625,4 +625,19 @@ function ($block_content, $block): string {
     10,
     2
 );
-// phpcs:enable Generic.Files.LineLength.MaxExceeded
+
+// Unset the Resistance Hub Campaign page when action_options is disabled
+// Remove it from selector within the editor
+add_filter('theme_page_templates', function($post_templates, $theme, $post) {
```

Comment:
I've added this filter to avoid to display the Resistance Hub option from selector when the feature is disabled. It will avoid incosistencies.